stat_fs.go

```diff
@@ -0,0 +1,66 @@
+package hdfs
+
+import (
+	"errors"
+
+	hdfs "github.com/colinmarc/hdfs/protocol/hadoop_hdfs"
+)
+
+var StatFsError = errors.New("Failed to get HDFS usage")
+
+// FsInfo provides information about HDFS
+type FsInfo struct {
+	capacity              uint64
+	used                  uint64
+	remaining             uint64
+	underReplicated       uint64
+	corruptBlocks         uint64
+	missingBlocks         uint64
+	missingReplOneBlocks  uint64
+	blocksInFuture        uint64
+	pendingDeletionBlocks uint64
+}
+
+func (c *Client) StatFs() (FsInfo, error) {
+	fs, err := c.getFsInfo()
+	if err != nil {
```

Comment:
Follow the pattern used in the other methods:

```
if err != nil {
	if nnErr, ok := err.(*rpc.NamenodeError); ok {
		err = interpretException(nnErr.Exception, err)
	}

	return nil, err
}
```

Also, this can just be one method.